docset/windows/vpnclient/Add-VpnConnection.md

```diff
@@ -159,7 +159,15 @@ SplitTunneling        : True
 ```
 
 This set of commands adds a VPN connection using a custom EAP authentication method.
-For more information about custom EAP authentication methods, see the New-EapConfiguration cmdlet.
+For more information about custom EAP authentication methods, see the **New-EapConfiguration** cmdlet.
+
+### Example 5: Add a VPN connection that uses already generated EAP XML configuration
+```powershell
+PS C:\> $EAPXml = [xml]Get-Content -Path C:\eap-config.xml
+PS C:\> Add-VpnConnection -Name "Test6" -ServerAddress "10.1.1.1" -TunnelType "L2tp" -EncryptionLevel "Required" -AuthenticationMethod Eap -SplitTunneling -AllUserConnection -RememberCredential -EapConfigXmlSteam $EAPXml
+```
+
+This set of commands adds a VPN connection using already generated EAP XML configuration. For more information about how to generate EAP XML configuration, see [EAP Configuration](https://docs.microsoft.com/windows/client-management/mdm/eap-configuration).
```

Comment:
```suggestion
This set of commands adds a VPN connection using an already generated EAP XML configuration. For more information about how to generate EAP XML configuration, see [EAP Configuration](https://docs.microsoft.com/windows/client-management/mdm/eap-configuration).
```
Just a small suggestion. I approve after @get-itips and @JohanFreelancer9 details.